karyon3-api/src/main/java/com/netflix/karyon/annotations/Priority.java

```
@@ -0,0 +1,21 @@
+package com.netflix.karyon.annotations;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Use Priority with classes that are auto-loaded (such as AutoBinder) to control
+ * the order in which these auto-loaded classes are called.  
```

I think we should throw an error.